packages/lib-webpack/package.json

```diff
@@ -24,7 +24,7 @@
     "api-extractor": "yarn run -T api-extractor -c $INIT_CWD/api-extractor.json"
   },
   "peerDependencies": {
```

Comment:
@vojtechszocs Since this needs to be published ASAP - do you want to bump the versions in package.json to 2.0.0 for the effected packages and publish after this PR merges?

Reply:
Let's do the package version bump for `lib-core` and `lib-webpack` in a separate PR and publish once it's merged.